test/OpenTelemetry.Exporter.Geneva.Tests/GenevaLogExporterTests.cs

```diff
@@ -1206,7 +1209,12 @@ private void AssertFluentdForwardModeForLogRecord(GenevaExporterOptions exporter
                 {
                     Assert.Equal(item.Value.ToString(), mapping["body"]);
                 }
-                else if (exporterOptions.CustomFields == null || exporterOptions.CustomFields.Contains(item.Key))
+                else if (
+                    exporterOptions.CustomFields == null ||
+                    exporterOptions.CustomFields.Contains("*") ||
+                    exporterOptions.CustomFields.Contains(item.Key) ||
+                    exporterOptions.CustomFields.Contains(partAName + "/" + "*") ||
+                    exporterOptions.CustomFields.Contains(partAName + "/" + item.Key))
```

Comment:
I guess this is going to cause tons of allocations on the hot path.

Reply:
This is for tests. Hot Path code uses hashsets and is strictly zero allocations